share/static/js/functions.js

```diff
@@ -52,6 +52,26 @@ function wait(ms) {
 	};
 }
 
+// 日付を見て自動的に告知メッセージを表示する
+// https://github.com/perl-entrance-org/perl-entrance-org.github.com/issues/93
+$(document).ready(function() {
+	"use strict";
+	const now = new Date().toLocaleString({ timeZone: 'Asia/Tokyo' });
+	$('.date').map((_,dataDom) => $(dataDom)).each((_, $dateDom) => {
+		const nextEventDate = $dateDom.data('date') !== undefined ? 
+			new Date($dateDom.data('date')).toLocaleDateString({ timeZone: 'Asia/Tokyo'})
+			: undefined;
```

Comment:
メモ（修正不要で大丈夫です）：

簡素化として

```suggestion
		const nextEventDate = $dateDom.data('date') && new Date($dateDom.data('date')).toLocaleDateString({ timeZone: 'Asia/Tokyo'});
```

もありかな。

`===` は undefined との比較において安全だと思うけれど、未定義との比較は違和感がある派にとって安心かも。

Reply:
なるほど確かに `&&`使うとスッキリっていう感じですね